apprepository.jsonnet

```diff
@@ -60,9 +60,9 @@ local labels = {
           serviceAccountName: $.serviceaccount.metadata.name,
           containers_+: {
             default: kube.Container("controller") {
-              image: "kubeapps/apprepository-controller@sha256:2adb4f419d6f63952f1f678dc14bab73c25b355b85857748172ee917135e7c38",
+              image: "kubeapps/apprepository-controller:v0.3.0",
```

Comment:
switching from sha to a tag here because we want to use the same version tag for all of the kubeapps core components (excludes vendored components like kubeless, nginx, mongo, etc.)